spec/models/article_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe Article, type: :model do
+  feature 'admin creates an article' do
+
+    scenario 'they see the article on the page' do
```

Comment:
This isn't actually doing anything, I'd remove these 3 lines, also not a concern for model specs